deepchecks/checks/performance/model_inference_time.py

```diff
@@ -0,0 +1,84 @@
+"""The model inference time check module."""
+import typing as t
+import timeit
+
+from deepchecks import SingleDatasetBaseCheck, CheckResult, Dataset, ConditionResult
+from deepchecks.utils import model_type_validation, DeepchecksValueError
+from deepchecks.string_utils import format_number
+
+
+__all__ = ['ModelInferenceTimeCheck']
+
+
+MI = t.TypeVar('MI', bound='ModelInferenceTimeCheck')
+
+
+class ModelInferenceTimeCheck(SingleDatasetBaseCheck):
+    """Measure model average inference time (in seconds) per sample."""
+
+    NUMBER_OF_SAMPLES: t.ClassVar[int] = 1000
```

Comment:
Shouldn't this appear as a check parameter in the __init__?

Reply:
updated